### DISCUSSION TITLE
Release author is incorrect in feed

### DISCUSSION THREAD

**Discussion by Vivelin** [community/community#18555] [Other Feature Feedback, Questions, & Ideas]
I own a repository, and I assigned someone as a contributor. If they publish a release, it shows up with me as the author of the release in the feed:

![Screenshot of the feed that reads "For You (Beta), Vivelin published a release"](https://user-images.githubusercontent.com/1766841/173509231-2a977175-236a-4b56-89a6-382010a220a6.png)

The [release itself](https://github.com/Vivelin/SMZ3Randomizer/releases/tag/v5.1.2) shows the correct author:

![Screenshot of the release that reads "MattEqualsCoder released this 17 hours ago"](https://user-images.githubusercontent.com/1766841/173509338-1ae85a17-f554-470d-b92c-74f1f6f20e85.png)

**Comment by MathiasReker** [comment]
please report it here: https://support.github.com/contact/bug-report

**Reply by Vivelin** [reply]
I did this, and they replied to me that the public feedback discussion board is the best place to report issues.
